custom_components/elevenlabs_tts/elevenlabs.py

```diff
@@ -134,45 +144,61 @@ async def get_tts_options(
         if not options:
             options = {}
 
+        if options.get(ATTR_AUDIO_OUTPUT, "mp3") != "mp3":
+            raise ValueError("Only MP3 output is supported.")
+
         # Get the voice from options, or fall back to the configured default voice
-        voice = options.get(
-            CONF_VOICE, self.config_entry.options.get(CONF_VOICE, DEFAULT_VOICE)
+        voice_opt = (
+            options.get(CONF_VOICE)
```

Comment:
Did we just get lucky that our `CONF_VOICE`'s value "voice" is the same name of the option that the assistant pipeline sets when a particular voice is specified in the assistant settings?

If that's the case, should we import that voice const that the assistant pipeline uses instead of redeclaring it ourselves (ensuring it'll always match)?

Reply:
we did indeed get lucky. I've removed conf_voice and use the attr in tts